adapter/resources/k8s-artifacts/choreo-connect/adapter-deployment.yaml

```diff
@@ -0,0 +1,179 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
Could you please update license header files of all the files you have added with this PR?